initScripts/aarch32/Ubuntu_16.04/Docker_18.03.sh

```diff
@@ -483,6 +502,9 @@ main() {
   trap before_exit EXIT
   exec_grp "fetch_cexec"
 
+  trap before_exit EXIT
```

Comment:
I think `remove_genexec` is unnecessary in this flow, but maybe we want to remove it all together in all the scripts in a different issue.